apps/src/userHeaderStatsigReporter/userHeaderStatsigReporter.js

```diff
@@ -1,6 +1,18 @@
 import statsigReporter from '../lib/util/StatsigReporter';
 
-document.addEventListener('DOMContentLoaded', function () {
+if (document.readyState !== 'loading') {
+  console.log('Document is ready');
+  runStatsigReporter();
+} else {
+  document.addEventListener('DOMContentLoaded', function () {
+    console.log('Document is loading');
```

Comment:
nit: Is this accurate? The document content has loaded at this point, yeah?

Reply:
The console says it's `interactive` so technically it's still loading? 😅 I can remove this if you think it's superfluous!

Reply:
According to [this documentation(https://developer.mozilla.org/en-US/docs/Web/API/Document/readyState#interactive), the document has loaded but subresources haven't. I think my brain doesn't like the word "loaded" on one line followed by "loading" on the next line. I think my preference would be to either remove this line or change it to say "DOM Content Loaded" or something similar. Presumably this line is just to denote when the listener starts?

Reply:
That makes sense! I'll update it to say "DOM content has loaded"